SmartDeviceLink_Example/ButtonManager.swift

```diff
@@ -0,0 +1,159 @@
+//
+//  ButtonManager.swift
+//  SmartDeviceLink
+//
+//  Created by Nicole on 4/11/18.
+//  Copyright © 2018 smartdevicelink. All rights reserved.
+//
+
+import Foundation
+import SmartDeviceLink
+import SmartDeviceLinkSwift
+
+typealias refreshUIHandler = (() -> Void)
```

Comment:
Types should start with a capital letter

Reply:
Fixed